Examples/PresenterExample/PresenterExampleViewController.swift

```diff
@@ -27,7 +27,7 @@ class PresenterExampleViewController: CollectionViewController {
         view.layer.cornerRadius = 5
         view.clipsToBounds = true
       },
-      layout: WaterfallLayout<UIImage>(columns:2, insets: bodyInset, axis: .horizontal),
+      layout: InsetLayout(InverseLayout(WaterfallLayout(columns:2)), insets: bodyInset),
```

Comment:
Make `CollectionLayout` have an instance method `func inverse() -> CollectionLayout<Data>`.

Inside that method, we created an inverseLayout and return it.

Same thing for `InsetLayout`.

So that we can have something like `WaterfallLayout(columns:2).inverse().inset(insets: bodayInset)`, which is more swifty.